libraries/standard/mqtt/utest/mqtt_lightweight_utest.c

```diff
@@ -161,7 +161,7 @@ static int32_t mockReceive( NetworkContext_t context,
     size_t bytesRead = 0;
 
     /* Treat network context as pointer to buffer for mocking  */
-    mockNetwork = ( *( uint8_t ** ) context );
+    mockNetwork = ( *( uint8_t ** ) pNetworkContext->buffer );
```

Comment:
`pNetworkContext->buffer` is now already a `uint8_t**`, so this cast is unnecesary:
```suggestion
    mockNetwork = *( pNetworkContext->buffer );
```